doc/source/visualizing/plots.rst

```diff
@@ -1054,6 +1055,68 @@ change the property of a single line, give also the index of the profile.
     # change only the first line
     plot.set_line_property("linestyle", "--", 0)
 
+.. _how-to-1d-unstructured-mesh:
+
+1D Line Sampling
+----------------
+
+YT has the ability to sample datasets along arbitrary lines
+and plot the result. You must supply five arguments to the ``LinePlot``
+class. They are enumerated below:
```

Comment:
That's exactly right.  I think what Britton is getting at is that superficially they might appear similar, but they are not.

No, `ray.profile` is not used for `LinePlot`. A ray data object is used internally though.

The big difference I'd highlight is that rays return data elements that intersect with the ray and have no guarantee about data spacing.